devel/darwin-xtools/Portfile

```diff
@@ -3,11 +3,14 @@
 PortSystem                  1.0
 PortGroup                   cmake 1.1
 PortGroup                   github 1.0
+PortGroup                   muniversal 1.0
 
 if {${os.arch} eq "powerpc"} {
     set xtools_version      6f6c04e8b25497851f6a5979a6e96023fffe22df
     github.setup            iains darwin-xtools ${xtools_version}
     version                 2.2.4
+    revision                1
+    epoch                   1
```

Comment:
Just curious, did you intend to set `epoch` to 1, for PPC?

Reply:
@mascguy yes, because @barracuda156 downgraded that port on PPC and forgot to increase epoch